### PR TITLE
Orientation

### PR DESCRIPTION
**Pull request by klauspdoll**

### CODE REVIEW

File: ProjectWindow.py [hunk 1]
```diff
@@ -365,6 +370,8 @@ def setup_guided_pane(self):
 
     def refresh_guided_pane(self):
         if self.trace: print('refresh_guided_pane')
+        if 'orientation' in self.input_specification:
+            self.guided_combo_orientation.setCurrentText(self.input_specification['orientation'])
```
Comment (pjknowles) [inline ProjectWindow.py]: I suggest
```
--- a/ProjectWindow.py
+++ b/ProjectWindow.py
@@ -370,8 +370,7 @@ class ProjectWindow(QMainWindow):

     def refresh_guided_pane(self):
         if self.trace: print('refresh_guided_pane')
-        if 'orientation' in self.input_specification:
-            self.guided_combo_orientation.setCurrentText(self.input_specification['orientation'])
+        self.guided_combo_orientation.setCurrentText(self.input_specification['orientation'] if 'orientation' in self.input_specification else list(molpro_input.orientation_options.keys())[0])
         if 'method' in self.input_specification:
             base_method = re.sub('[a-z]+-', '', self.input_specification['method'], flags=re.IGNORECASE)
             prefix = re.sub('-.*', '', self.input_specification['method']) if base_method != self.input_specification[
```
to cover the case that orientation is first set to charge, and then manually deleted from the input

Comment (pjknowles) [reply ProjectWindow.py]: It using the convention that the first option in the dictionary is the one that Molpro will adopt if nothing is given in the input. It's a bit flaky, but I can't think of a better way right now.

**Review by pjknowles — CHANGES_REQUESTED** [review]
It looks good after the one suggested improvement

File: ProjectWindow.py (hunk 1 above)
Comment (klauspdoll) [reply ProjectWindow.py]: Done now, with f8f8ee5477

**Review by pjknowles — APPROVED** [review]

**Pull request merged**